src/Site_Command.php

```diff
@@ -665,14 +991,16 @@ private function create_etc_hosts_entry() {
 	private function install_wp() {
 		EE::log( "\nInstalling WordPress site." );
 		chdir( $this->site_root );
-		$install_command = "docker-compose exec --user='www-data' php wp core install --url='" . $this->site_name . "' --title='" . $this->site_title . "' --admin_user='" . $this->site_user . "'" . ( ! $this->site_pass ? "" : " --admin_password='" . $this->site_pass . "'" ) . " --admin_email='" . $this->site_email . "'";
+		$install_command = "docker-compose exec --user='www-data' php wp core install --url='$this->site_name' --title='$this->site_title' --admin_user='$this->site_user'" . ( ! $this->site_pass ? "" : " --admin_password='$this->site_pass'" ) . " --admin_email='$this->site_email'";
 
-		EE::debug( 'COMMAND: ' . $install_command );
-		EE::debug( 'STDOUT: ' . shell_exec( $install_command ) );
+		$core_install = \EE\Utils\default_launch( $install_command );
+		if ( ! $core_install ) {
+			EE::warning( 'WordPress install failed. Please check logs.' );
+		}
 
 		if ( 'wpsubdom' === $this->site_type || 'wpsubdir' === $this->site_type ) {
-			$type               = $this->site_type === 'wpsubdom' ? ' --subdomains' : '';
-			$multi_type_command = "docker-compose exec --user='www-data' php wp core multisite-convert" . $type;
+			$type               = $this->site_type === 'wpsubdom' ? '--subdomains' : '';
+			$multi_type_command = "docker-compose exec --user='www-data' php wp core multisite-convert $type";
```

Comment:
Why you're first creating a single site and then migrating it to subdir/subdomain multi site? Why we are not using `wp core multisite-install` for this ?
ref: https://developer.wordpress.org/cli/commands/coe/multisite-install/
This function is for fresh WP install right?